xAPI.md

```diff
@@ -255,6 +255,25 @@ learning that SCORM could not enable.
 
 ## 1.0 Summary of the Experience API
 
+The Experience API (Application Programming Interface) - or xAPI as it is commonly called - is a framework that 
+facilitates the description of learning experiences and their communication and storage in a consistent manner 
+in a digital environment. As an application programming interface, the xAPI does not set out how the framework  
```

Comment:
```As an application programming interface, the xAPI does not set out how the framework should be implemented. Rather, it sets out the essential components of that framework and how they relate to each other.``` - I *think* I get what this is trying to say, that things like database structure are out of scope, and agree this is an important point, but I don't think this is clear at all as it is.